src/color-picker/color-picker.js

```diff
@@ -145,6 +158,11 @@ const Self = class ColorPicker extends ColorElement {
 				return [this._el.sliders, this._el.swatch];
 			},
 		},
+		spacechange: {
+			from () {
+				return this._el.space_picker;
```

Comment:
Hm that will make it hard to slot custom elements, so I'd revert back to glue code

Reply:
Done!